### ISSUE TITLE
Let the user confirm installation of each app after they saw app's size

### ISSUE THREAD

**Issue by farooqkz** [strukturart/bHacker-store-client#57]

**Comment by strukturart** [comment]
we don't know the size of the app

**Comment by strukturart** [comment]
in this ann I can only display what is in the database. if you want blue water gushing out of the spring you have to dig deep in the mountain.

**Comment by farooqkz** [comment]
We can ask from the HTTP server if it tells us about size. Want me to send a PR?

**Comment by farooqkz** [comment]
@strukturart If you agree on adding this feature, I can send a PR.

**Comment by farooqkz** [comment]
Why this was closed? Did you implement this thing in the store app?

**Comment by strukturart** [comment]
I've been waiting for you to send me something ...... not come ...... issue closed

**Comment by farooqkz** [comment]
@strukturart An issue requesting a feature may be closed only when the developer(s) reject adding that feature or when the feature is added. It seems that you are not rejecting so I reopen this issue for now till someone(me or you or anyone else) add this feature.

Also you have waited for me without informing me about this matter and at the same time I was waiting for an approval from you!

**Comment by strukturart** [comment]
I think the information about the file size should be in the database and not the task of the client to determine it.

https://gitlab.com/banana-hackers/store-db/-/blob/master/_generator/index.js

**Comment by farooqkz** [comment]
It's not in the DB but why don't we inform the user about file size like any other downloader?

**Comment by strukturart** [comment]
Informing the user is the task of the client, I can do that.

**Comment by farooqkz** [comment]
Do you know how should you do that?

**Comment by strukturart** [comment]
please adapt the generator file so that the file size is determined

**Comment by farooqkz** [comment]
Nope. Let me do it myself. Give me a few days but less than a week :)

**Comment by strukturart** [comment]
hmh, if it is in the database, other clients can also use the information

**Comment by farooqkz** [comment]
Let's discuss this in Discord so that others can participate

**Comment by strukturart** [comment]
@Simon-Laux pls add the filesize into the store db

**Comment by farooqkz** [comment]
He has said that such a thing won't be added!

**Comment by strukturart** [comment]
okay, that would be the right place in the db so that the information is available to all clients. @farooqkz  could you send me a snippet that i do it in the client

**Comment by farooqkz** [comment]
https://bitexperts.com/Question/Detail/3316/determine-file-size-in-javascript-without-downloading-a-file

**Comment by Simon-Laux** [comment]
> @Simon-Laux pls add the filesize into the store db

read the discord backlog we discussed this there.
